src/handlers/initiateDataRequest.ts

```diff
@@ -63,3 +84,20 @@ const handleInvalidSignature = async () => {
     })
   }
 }
+
+const handleUnmatchedRequest = async (requestDetails: DataRequestParams) => {
```

Comment:
Would it be tidier to just pass in the zendeskId rather than the whole requestParam object?

Reply:
Done!